packages/babel-parser/src/parser/expression.js

```diff
@@ -1544,7 +1544,8 @@ export default class ExpressionParser extends LValParser {
     return parenExpression;
   }
 
-  shouldParseArrow(): boolean {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
It is better to add the comment like below:
```suggestion
  // eslint-disable-next-line no-unused-vars -- `params` is used in typescript plugin
```